### ISSUE TITLE
Shadowcaster - Combat music issue

### ISSUE THREAD

**Issue by Krusher97** [joncampbell123/dosbox-x#1550]
**Describe the bug**
In Shadowcaster when you attack an enemy(or an enemy attacks you) combat music should start playing until you kill the enemy or run away. In Dosbox, only the first few notes of the music are played and the rest is cut off, causing the first few notes to repeat every time you attack or get attacked by an enemy. 
I have tested the Shareware release, the Floppy disk version and the CD version. All of them have this issue.

**To Reproduce**
1. Run the installer to set up sound 
2. Start the game
3. Disable sound effects in the menu
4. Attack the first enemy

**Environment (please complete the following information):**
 - Windows 7 64 bits
 - DOSBox-X 0.83.1
 - Default configurations

**Comment by Allofich** [comment]
I can confirm. It only happens with auto/dynamic core, though.

**Comment by Allofich** [comment]
@Krusher97, try playing with Normal core.

**Comment by Krusher97** [comment]
Yeah it works properly with core set to normal. Should this be closed then?

**Comment by joncampbell123** [comment]
@Allofich Full core is deprecated? I'm happy to drop it from DOSBox-X as well. I've never used or maintained it and as far as I know nobody else using DOSBox-X has either.

The DOSBox-X version of full core has some incomplete guest page fault fixes for Windows 95 that still has a few issues I was never able to resolve.

**Comment by Allofich** [comment]
> Full core is deprecated?

I guess so. The DOSBox wiki site where that is written seems to be the official wiki, and I see at least one of the current DOSBox devs having contributed to page edits there.

If its deprecated it seems OK to drop it to me. You could always put it back if someone complains they were using it for something. I also don't really see much point of the "simple" core (supposedly optimized for older real-mode games, which I would think would all run fine with normal core) but maybe it is useful for something.

**Comment by joncampbell123** [comment]
full core is a completely different implementation.

simple core is just a different wrapper that includes the same disassembler/emulation code as normal core, which makes it much easier to maintain.

Simple core's appeal is that it could possibly provide a slight performance boost by omitting protected mode paging. However on today's machines the boost might be so slight that it probably no longer helps and just adds bloat to the executable.

I am all for dropping full core.
